quarkus/service/src/main/resources/application.properties

```diff
@@ -82,7 +82,9 @@ polaris.features.defaults."SUPPORTED_CATALOG_STORAGE_TYPES"=["S3","GCS","AZURE",
 # polaris.features.realm-overrides."my-realm"."INITIALIZE_DEFAULT_CATALOG_FILEIO_FOR_TEST"=true
 # polaris.features.realm-overrides."my-realm"."SKIP_CREDENTIAL_SUBSCOPING_INDIRECTION"=true
 
-polaris.persistence.type=in-memory
+polaris.persistence.type=eclipse-link
```

Comment:
Not sure about using `eclipse-link` by default. We can always set it in Helm values, but running from the jar locally and having to override it back to in-memory (most of the time) seems inconvenient.

Reply:
See my previous comment. 

You shouldn't need to switch to `in-memory` when running locally. H2 is configured with an in-memory database, so in the end, it's the same from a UX perspective.

Reply:
Ah, I missed H2 in-memory... I assumed it was file-based :facepalm:

Reply:
H2 in-memory and the TreeMap metastore do not have the same semantics. For example, the in-memory metastore currently prints credentials when it bootstraps.

Reply:
Exactly, and you don't want your credentials printed when running in production right? These settings are the production ones.

For running locally, we can instruct users to switch to in-memory:

```shell
 docker run -it -p 8181:8181 -e polaris.persistence.type=in-memory apache/polaris
```

Then the credentials get printed to the console.